traefik/templates/_helpers.tpl

```diff
@@ -37,7 +37,7 @@ If release name contains chart name it will be used as a full name.
 app.kubernetes.io/name: {{ template "traefik.name" . }}
 helm.sh/chart: {{ template "traefik.chart" . }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
-app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/instance: {{ .Release.Name }}-{{ .Release.Namespace }}
```

Comment:
Could this potentially break something? I also do not see why it needs a suffix as the (helm) release name should be already unique per namespace.

Reply:
According to the Kubernetes documentation this label should contain "A unique name identifying the instance of an application". So for the global objects `ClusterRole` and `ClusterRoleBinding` this would not be unique without the suffix if there were multiple releases with the same name but in different namespaces. I do agree though that the suffix is not necessary for the namespaced objects, which is why I initially used a separate label helper for `ClusterRole` and `ClusterRoleBinding`, but maybe it is nice to make this label consistent throughout all objects like @mloiseleur suggested.

Reply:
@LorianColtof Discussing this change on `app.kubernetes.io/instance` label, we see potentially a breaking change here and we are not sure of the use case. Can you detail us more why you also need this change ? or do you think we are good to merge this PR without this label change ?

Reply:
I don't need this label change in particular. I added it just to keep this label unique for the `ClusterRole`/`ClusterRoleBinding` cases.

Reply:
@LorianColtof Thanks. It's more clear. We bumped to a Major version, since it can breakj something for people using this label.